bosonprotocol/core-components
# feat(widgets): updated widget theme

## Description

Widget theme was altered to be darker.

## How to test

Run `npm ci && npm run build && npm run dev`, navigate to `localhost:4000` and click "Submit". Observe the updated widget theme (includes the modals).
